Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -43,7 +43,7 @@ def gistify_logs(f)
       puts <<~EOS
         You can create a new personal access token:
          #{GitHub::ALL_SCOPES_URL}
-        and then set the new HOMEBREW_GITHUB_API_TOKEN as the authentication method.
+        #{Utils::Shell.set_variable_in_profile("HOMEBREW_GITHUB_API_TOKEN", "YOUR_TOKEN")}
```

Comment:
I think `$YOUR_TOKEN` would be more obvious, perhaps?

Reply:
I tried that initially but I'm having issues with `Utils::Shell.sh_quote` adding a backslash before the `$`. Any pointers?

result of passing `$YOUR_TOKEN`:
`'export HOMEBREW_GITHUB_API_TOKEN=\$YOUR_TOKEN'`

Reply:
In that case perhaps a lowercase `your_token_here` or similar would be better?